src/components/tool_tip/tool_tip.test.tsx

```diff
@@ -54,4 +54,19 @@ describe('EuiToolTip', () => {
     await sleep(260); // wait for showToolTip setTimout
     expect(takeMountedSnapshot(component)).toMatchSnapshot();
   });
+
+  test('applies our block stylings', async () => {
```

Comment:
Just a small tweak on wording that aligns to our conventions. We tend not to use sentence style grammar or use pronouns like "our". So something more succinct like:

```suggestion
  test(`display prop renders 'block'`, async () => {
```